ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -647,6 +713,7 @@ public PutChunk() {
      */
     void clear() {
       chunkIndex = -1;
+      chunkBlobSize = -1;
```

Comment:
shouldn't `chunkUserMetadata` and `encryptedPerBlobKey` be cleared too?

Reply:
I have fixed encryptedPerBlobKey. Chunk User-metadata is already set to orig user-metadata in the same method.